opslevel/resource_opslevel_integration_azure_resources.go

```diff
@@ -0,0 +1,223 @@
+package opslevel
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/path"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/opslevel/opslevel-go/v2024"
+)
+
+var _ resource.ResourceWithConfigure = &IntegrationAzureResourcesResource{}
+
+var _ resource.ResourceWithImportState = &IntegrationAzureResourcesResource{}
+
+func NewIntegrationAzureResourcesResource() resource.Resource {
```

Comment:
I think there is a double `Resources` on all of these things - cross referencing the AWS one it seems like maybe a copy and paste replace operation gone wrong?

Reply:
We should bring this up somewhere because I don't think thats a great choice.  Marketing is talking about is as the `Azure Integration`

Reply:
https://jk-labs.slack.com/archives/C05RNQ6FD8U/p1720453412988499